src/systems/systems/src/pressurization/cabin_pressure_controller.rs

```diff
@@ -0,0 +1,475 @@
+                    self.cabin_pressure = Pressure::new::<hectopascal>(
+                        -0.0002 * ambient_pressure.powf(2.) + 0.5463 * ambient_pressure + 658.85,
+                    );
+                }
+                self.cabin_target_vs = Velocity::new::<foot_per_minute>(0.);
+            }
+            PressureSchedule::TakeOff => {
+                if self.cabin_delta_p() < Pressure::new::<psi>(0.1) {
+                    self.cabin_target_vs = Velocity::new::<foot_per_minute>(-400.)
+                } else {
+                    self.cabin_target_vs = Velocity::new::<foot_per_minute>(0.)
+                }
+            }
+            PressureSchedule::ClimbInternal => {
+                if self.cabin_delta_p() >= Pressure::new::<psi>(8.06) {
+                    self.cabin_target_vs = Velocity::new::<foot_per_minute>(750.);
+                } else if self.cabin_altitude() >= Length::new::<foot>(8050.) {
+                    self.cabin_target_vs = Velocity::new::<foot_per_minute>(0.);
+                } else if self.cabin_delta_p() < Pressure::new::<psi>(8.06) {
+                    // Formula based on existing graphs and tables to simulate climb schedule
+                    self.cabin_target_vs = Velocity::new::<foot_per_minute>(
+                        self.aircraft_vs.get::<foot_per_minute>()
+                            * (0.00000525 * context.indicated_altitude().get::<foot>() + 0.09),
+                    );
+                }
+            }
+            PressureSchedule::Cruise => {
+                self.cabin_target_vs = Velocity::new::<foot_per_minute>(0.);
+            }
+            PressureSchedule::DescentInternal => {
+                let target_vs = Velocity::new::<foot_per_minute>(
+                    self.get_int_diff_with_ldg_elev().get::<foot>()
+                        * self.aircraft_vs.get::<foot_per_minute>()
+                        / self.get_ext_diff_with_ldg_elev(context).get::<foot>(),
+                );
+                if target_vs <= Velocity::new::<foot_per_minute>(-750.) {
+                    self.cabin_target_vs = Velocity::new::<foot_per_minute>(-750.);
+                } else if target_vs >= Velocity::new::<foot_per_minute>(500.) {
+                    self.cabin_target_vs = Velocity::new::<foot_per_minute>(500.);
+                } else {
+                    self.cabin_target_vs = target_vs;
+                }
+            }
+            PressureSchedule::Abort => {
+                if self.cabin_altitude() < self.departure_elevation() - Length::new::<foot>(187.818)
+                {
+                    self.cabin_target_vs = Velocity::new::<foot_per_minute>(500.)
+                } else if self.cabin_altitude()
+                    == self.departure_elevation() - Length::new::<foot>(187.818)
+                {
+                    self.cabin_target_vs = Velocity::new::<foot_per_minute>(0.)
```

Comment:
😄 This `187.818` number represents the altitude in feet equivalent to 0.1 PSI delta pressure at sea level. According to the FCOM, the Airbus pressurizes the cabin so that when it lands it's not at the exact same pressure as ambient but at 0.1 PSI difference to avoid a pressure surge.

I've added a `const` name for it now that should hopefully add clarity in the code.